src/doc/rustdoc/src/what-is-rustdoc.md

```diff
@@ -37,7 +37,7 @@ top, with no contents.
 ## Configuring rustdoc
 
 There are two problems with this: first, why does it
-think that our package is named "lib"? Second, why does it not have any
+think that our crate is named "lib"? Second, why does it not have any
 contents?
```

Comment:
It is not immediately clear to me that this is a typo, as Rust has packages and packages are not crates, see https://doc.rust-lang.org/book/ch07-01-packages-and-crates.html

Reply:
https://github.com/rust-lang/rust/pull/112741/files#diff-3a33d0f38f7a3cd5deb856ff984abfbcdcdb5adc9203734e739a1572ca2a3a1fR34

According to the description of the context, I think it is more appropriate to use `crate`.

Reply:
Oh, well-spotted!